build.yaml

```diff
@@ -16,6 +16,8 @@ targets:
           dart2js_args:
           - --minify
           - --fast-startup
+          - -DPRE_NULL_SAFETY_SERVER_URL=https://v1.api.dartpad.dev/
```

Comment:
Do you just want to put these in as defaults in the variable definitions?

Reply:
I could. Putting them in here means its possible to run build_runner separate from grinder. I don't think that's a known workflow, but kind of weird that build_runner could not be run by itself without these args.